### PR TITLE
🔧 fix schema default value when AOT is off

### PR DESCRIPTION
**Pull request by macabeus**
This PR fixes https://github.com/elysiajs/elysia/issues/950

I could not fix the default value for cookies but, as far as I checked, it already doesn't work when AOT is on, so I think that this PR is good enough for now. At least it ensures that the behavior when AOT is on or off is even.

### CODE REVIEW

File: test/core/elysia.test.ts [hunk 1]
```diff
@@ -173,4 +173,37 @@ describe('Edge Case', () => {
 		// @ts-expect-error private property
 		expect(main.getGlobalRoutes().length).toBe(2)
 	})
+
+	describe('value returned from transform has priority over the default value from schema', () => {
```
Comment (macabeus) [inline test/core/elysia.test.ts:177]: This new test isn't exactly related to this issue, but I added it because I could not find a test ensuring this behavior.
